src/test/37-billing.ts

```diff
@@ -0,0 +1,86 @@
+import { expect } from "chai";
+import { C8Client } from "../jsC8";
+
+const C8_VERSION = Number(process.env.C8_VERSION || 30400);
+
+
+describe("validating billing apis", function () {
+    this.timeout(60000);
+
+    let c8Client: C8Client;
+    const testUrl: string =
+        process.env.TEST_C8_URL || "https://test.macrometa.io";
+    const tenantName = "ajay.krishna_macrometa.com";
```

Comment:
not your name.

Reply:
we create somewhere using some date please check.

Reply:
updated